docker-image-onnx/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM ubuntu:24.04
+RUN apt update && apt install -y python3 python3-pip
+RUN pip config set global.break-system-packages true
+RUN pip install poetry
+WORKDIR /app
+ADD docker-image-onnx/ .
```

Hadolint (2.14.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)